CHANGELOG.rst

```diff
@@ -2,7 +2,7 @@
 Changelog
 =========
 
-2.0.0 "Règâh" (2022-??-??) - in testing
+2.0.0 "Règâh" (2022-10-??) - in testing
```

Comment:
Testing can be gone right?

Also, sorry no possibility to comment on that line, I feel like there should be a summary of the upgrade procedure:

1. Update to 1.1.6
2. Run management command 1 and fix issues
3. Run management command 2 and fix issues
4. If all issues solved, upgrade to 2.0.0

Reply:
I thought we had covered that with the "manual interventions required", but probably we can reword that a bit.

W/r to the "in testing" - as long as 2.0.0 final is not out yet, it's still in testing. We still need to fix a couple other bugs and then re-test before we can release.